INSTALLATION.md

```diff
@@ -77,14 +77,19 @@ Congratulations! You have successfully installed **Massa Station** on your compu
 2. Open your file manager and navigate to the location where the `.deb` package is saved.
 3. The package manager will launch and display **Massa Station** installation page. 
 4. Review the package information and dependencies, if any, and click on the red "Delete" button.
-5. Proceed to GUI installation starting at step 2. 
-
+5. Proceed to GUI installation starting at step 2.
 
 ### Manually install the HTTPS certificate
-1. Open your browser "Manage Certificates" settings (e.g. chrome://settings/certificates or brave://settings/certificates) and switch to the "Authorities" tab.
-2. Click "Import", locate the Massastation certificate at **/etc/massastation/certs/rootCA.pem** and import it to the browser.
 
+To enable Massa Station to function with HTTPS on an unsupported browser, you will need to manually install the HTTPS certificate authority. Follow these steps to import the certificate authority into your browser:
 
+1. Locate the certificate authority file at the following path:
+   **/etc/massastation/certs/rootCA.pem** on Linux and MacOS,
```

Comment:
I tried to follow this instruction but I don't know where to locate and how. Where do i search for /rootCA.pem ?

Reply:
the rootCA.pem is located at the path: `/etc/massastation/certs/rootCA.pem`, is it this path that is not clear for you?

Reply:
I think the issue is more with "How to locate". We need to explain people to do the following:
1. Copy this path: /etc/massastation/certs/
2. Open finder
3. Click on the "Go" button at the top of your screen
4. Click on the "Go to folder..." button at the bottom of the menu
5. Paste the path you copied previously
6. Press enter

Users are not used to go to this kind of directories on MacOS since they are not easy to access from Finder